WooCommerce/Classes/ViewRelated/Coupons/CouponListViewController.swift

```diff
@@ -216,7 +255,7 @@ private extension CouponListViewController {
     }
 
     func configureNavigationBarItems(hasCoupons: Bool) {
-        navigationItem.rightBarButtonItems = hasCoupons ? [searchBarButtonItem] : []
+        navigationItem.rightBarButtonItems = hasCoupons ? [createCouponButtonItem, searchBarButtonItem] : [createCouponButtonItem]
```

Comment:
I forgot this part in the last review 😅 I think we should hide the coupon creation behind a feature flag, since this is still a work in progress. We also need to test this feature internally first so maybe it's safer not to show this to the public yet.